src/components/issues/IssuesPage.js

```diff
@@ -39,13 +39,13 @@ type OwnProps = {};
 
 class IssuesPage extends Component<ConnectedProps & OwnProps> {
   componentDidMount() {
-    const { name, repo } = this.props.match.params;
-    this.handleFetchIssues();
-    this.props.fetchCollaborators(name, repo);
-    this.props.fetchRepoAssignees(name, repo);
+    this.handleRequestByParams();
   }
 
   componentDidUpdate(prevProps) {
+    if (prevProps.location.pathname !== this.props.location.pathname) {
```

Comment:
why is this update based on url rather than an id for the issue that changed?

Reply:
Because its the issues page (not a single issue) so the only reference change is the url params (which by, it fetches the requested issues)